### DISCUSSION TITLE
How to build and import a local repo of MRTK into my Unity project

### DISCUSSION THREAD

**Discussion by ShayneRC** [microsoft/MixedRealityToolkit-Unity#11843] [General]
## Describe the issue
I'd like to make my own personal change to the MRTK staying on the version I am currently on (not upgrading to MRTKv3). I've forked the repo and made my change but I'm unsure on how to go about building it so I can use it as a package in Unity.

## Feature area
It would be great if there were steps available on how to get a local version of the MRTK into my Unity project without just dragging the assets in. Preferably how it's currently done with making each part into a .tgz file.

## Existing doc link
N/A

## Additional context
This would help for future custom development anyone else may want.

**Comment by AMollis** [comment]
Thank you for reporting this issue.  Unfortunately, we are only fixing high priority issues for MRTK2.  There is currently no active development on MRTK2.

Moving this issue to the forums.

If you need further assistance, please reach out to our support team at https://aka.ms/hldevsupport

**Reply by ShayneRC** [reply]
Hi Adam,

I believe this should be a priority as it would help developers fix or change things they want in the MRTK2 which should mean less issues being brought up. I'm not 100% sure but I don't think this would take long to do - it's just some documentation outlining how to modify and deploy your own version of the MRTK.
